src/modules/docker_context.rs

```diff
@@ -35,50 +42,57 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
     )
     .join("config.json");
 
-    if !docker_config.exists() {
-        return None;
-    }
+    let docker_context_env = context
+        .get_env_os("DOCKER_HOST")
+        .or_else(|| context.get_env_os("DOCKER_CONTEXT"));
```

Comment:
I think something like this would work better here
```rust
std::array::IntoIter::new(["DOCKER_HOST", "DOCKER_CONTEXT"]).find_map(context.get_env_os)
```